src/org/thoughtcrime/securesms/qr/QrCodeHandler.java

```diff
@@ -75,6 +75,15 @@ public void handleQrData(String rawString) {
                 builder.setNegativeButton(R.string.cancel, null);
                 builder.setCancelable(false);
                 break;
+            case DcContext.DC_QR_LOGIN:
```

Comment:
```suggestion
                break;

            case DcContext.DC_QR_LOGIN:
```

Reply:
mmmh it seems my recommendation to remove the empty spaces removed the whole blank line